### ISSUE TITLE
test of fits2bitmap (orientation) fails 

### ISSUE THREAD

**Issue by mhvk** [astropy/astropy#6960]
Not sure how this can escape our testing, but on my machine I got an error with numpy 1.14.x for the following, and then found it failed more generally:
```

self = <astropy.visualization.scripts.tests.test_fits2bitmap.TestFits2Bitmap object at 0x7fe19af6ec88>
tmpdir = local('/tmp/user/1000/pytest-of-mhvk/pytest-27/test_orientation0')

    def test_orientation(self, tmpdir):
        """
            Regression test to check the image vertical orientation/origin.
            """
    
        filename = str(tmpdir.join(self.filename))
        data = np.zeros((32, 32))
        data[0:16, :] = 1.
        fits.writeto(filename, data)
        main([filename, '-e', '0'])
    
        img = mpimg.imread(filename.replace('.fits', '.png'))
>       assert img[0, 0, 0] == 0.
E       assert 1.0 == 0.0

astropy/visualization/scripts/tests/test_fits2bitmap.py:70: AssertionError
```

**Comment by larrybradley** [comment]
@mhvk  Does #6959 fix this?

**Comment by mhvk** [comment]
No, it does not. I'm very surprised I see this failure on my machines, but it is not see on travis. Maybe it is a different matplotlib version? Mine is 2.0.0

**Comment by mhvk** [comment]
p.s. My change in #6959 was made while trying to investigate the failure - but I couldn't find a good reason!

**Comment by larrybradley** [comment]
It passes for me on `2.1.0`.  I'm not sure which version I had when I originally wrote the test, but something may have changed in `matplotlib`.  I will test older versions.

**Comment by larrybradley** [comment]
Fixed in #6969.
